impl/src/main/java/com/sun/faces/application/resource/ResourceImpl.java

```diff
@@ -188,16 +188,11 @@ public URL getURL() {
                 responseHeaders = new HashMap<>(6, 1.0f);
             }
 
-            long expiresTime;
             if (FacesContext.getCurrentInstance().isProjectStage(Development)) {
-                expiresTime = new Date().getTime();
+                responseHeaders.put("Cache-Control", "no-cache");
```

Comment:
Should be `no-store, must-revalidate`.

Reply:
Ok, I changed this.
IMO this is to some degree a matter of taste. Even englisch and german MDN - page for https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control prefer different solutions.

Reply:
Not a matter of taste. If the resource was already (accidentally) cached, `no-cache` doesn't wipe it.